docs/topics/api/overview.rst

```diff
@@ -254,17 +234,6 @@ Outgoing Links
 
 How outgoing links are treated has changed in v5 API.
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Default v3/v4 API behavior
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If the ``wrap_outgoing_links`` query parameter is present, any external links
-returned for properties such as ``support_url`` or ``homepage`` will be wrapped
-through ``outgoing.prod.mozaws.net``. Fields supporting some HTML, such as
-add-on ``description`` or ``summary``, always do this regardless of whether or
-not the query parameter is present.
-
-
 ^^^^^^^^^^^^^^^
 v5 API behavior
```

Comment:
As above, should this be changed to say "Default API behavior", because this doc is now about `v5`?

Reply:
https://github.com/mozilla/addons-server/pull/16303/commits/97c1654fb9e739eca3fd3ff3d297eaa054090a30